android/settings.gradle

```diff
@@ -1 +1 @@
-include ':app', ':trustwalletcore'
+include ':app', ':trustwalletcore', ':wallet-core-proto'
```

Comment:
let's change `trustwalletcore` to `wallet-core`? too?

Reply:
Fixed